WordPress/Classes/ViewRelated/Site Creation/WebAddress/WebAddressWizardContent.swift

```diff
@@ -305,7 +305,7 @@ final class WebAddressWizardContent: CollapsableHeaderViewController {
         view.addSubview(noResultsLabel)
 
         NSLayoutConstraint.activate([
-            noResultsLabel.widthAnchor.constraint(equalTo: headerView.widthAnchor),
+            noResultsLabel.widthAnchor.constraint(equalTo: headerView.widthAnchor, constant: CGFloat(-50)),
```

Comment:
Everything looked great in portrait, but I noticed that in landscape the constraint didn't respect the safe area so the notch was cutting off content. If we use the TableView for the width constraint we may not have this problem. Let me know your thoughts!

**Before**

<img width="972" alt="Screen Shot 2022-03-12 at 17 36 24" src="https://user-images.githubusercontent.com/2092798/158038592-fd5c01a5-8441-494d-ad97-d0ea9fbecdfd.png">


```suggestion
            noResultsLabel.widthAnchor.constraint(equalTo: table.widthAnchor, constant: -50),
```

![image](https://user-images.githubusercontent.com/2092798/158038561-c3c32dc2-9394-4faa-8dfe-84eef5bb001f.png)

**After**

<img width="972" alt="Screen Shot 2022-03-12 at 18 31 38" src="https://user-images.githubusercontent.com/2092798/158038623-a979c80c-4e8e-4edd-9571-8158e1f1f811.png">

Reply:
Awesome, thank you for the investigation and the suggestion @twstokes 🙇 !